pkgs/development/tools/fsautocomplete/default.nix

```diff
@@ -0,0 +1,53 @@
+{ buildDotnetModule, mkNugetDeps, emptyFile, lib, dotnet-sdk }:
+let
+  fsautocomplete = buildDotnetModule rec {
+    pname = "fsautocomplete";
+    version = "0.58.2";
+    src = emptyFile;
```

Comment:
Please use dontUnpack instead

Reply:
Thanks. Fixed all issues you've mentioned.